Simplified build process to reduce dependencies.
--- a/webpack.config.builder.js
+++ b/webpack.config.builder.js
@@ -3,8 +3,6 @@
 	path = require('path'),
 	webpack = require('webpack'),
 	devPath = path.resolve(__dirname, 'dev'),
-	CopyWebpackPlugin = require('copy-webpack-plugin'),
-	WebpackNotifierPlugin = require('webpack-notifier'),
 	loose = true;
 
 process.noDeprecation = true;
@@ -54,7 +52,7 @@
 					include: [devPath],
 					options: {
 						cacheDirectory: true,
-						presets: [['env', {
+						presets: [[{
 							loose: loose,
 							modules: false,
 							targets: {
