Bundled libraries missing from Debian.
--- a/dev/boot.js
+++ b/dev/boot.js
@@ -1,10 +1,8 @@
 
 import window from 'window';
 
-import {Promise} from 'es6-promise-polyfill/promise.js';
-import {progressJs} from '../node_modules/Progress.js/src/progress.js';
+import {progressJs} from '../debian/lib/progressjs/progress.js';
 
-window.Promise = window.Promise || Promise;
 window.progressJs = window.progressJs || progressJs();
 
 window.progressJs.onbeforeend(() => {
@@ -21,7 +19,6 @@
 	}
 });
 
-require('json3');
 require('../vendors/modernizr/modernizr-custom.js');
 require('Common/Booter');
 
--- a/webpack.config.builder.js
+++ b/webpack.config.builder.js
@@ -31,14 +31,17 @@
 			}),
 			new WebpackNotifierPlugin(),
 			new CopyWebpackPlugin([
-				{from: 'node_modules/openpgp/dist/openpgp.min.js', to: 'js/min/openpgp.min.js'},
-				{from: 'node_modules/openpgp/dist/openpgp.worker.min.js', to: 'js/min/openpgp.worker.min.js'}
+				{from: 'debian/lib/openpgp/dist/openpgp.min.js', to: 'js/min/openpgp.min.js'},
+				{from: 'debian/lib/openpgp/dist/openpgp.worker.min.js', to: 'js/min/openpgp.worker.min.js'}
 			])
 		],
 		resolve: {
 			modules: [devPath, 'node_modules'],
 			extensions: ['.js'],
 			alias: {
+				'pikaday$': __dirname + '/usr/lib/nodejs/pikaday/pikaday.js',
+				'js-cookie$': __dirname + '/debian/lib/js-cookie/js.cookie.js',
+				'classnames$': __dirname + '/usr/lib/nodejs/classnames/index.js',
 				'Opentip$': __dirname  + '/dev/External/Opentip.js',
 				'ko$': __dirname  + '/dev/External/ko.js'
 			}
